src/Contracts/SearchQuery.php

```diff
@@ -27,6 +27,7 @@ class SearchQuery
     private ?int $hitsPerPage;
     private ?int $page;
     private ?array $vector;
+    private ?HybridSearchOptions $hybrid;
```

Comment:
```suggestion
    private ?HybridSearchOptions $hybrid = null;
```

Reply:
I was testing it now too. and have made a similar commit. please check.

Reply:
You've commited it to another PR. so please choose one and close another one.